src/index.js

```diff
@@ -6,15 +6,23 @@ exports.register = function (server, options, next) {
   var client = new raven.Client(options.dsn, options.client);
   server.expose('client', client);
   server.on('request-error', function (request, err) {
+    var baseUrl = request.info.uri;
+    if (baseUrl === undefined) {
+      if (request.info.host) baseUrl = server.info.protocol + '://' + request.info.host;
+      else baseUrl = server.info.uri;
+    }
     (client.captureException || client.captureError).call(client, err, {
+      request: {
```

Comment:
Please sort these to match the Raven parser for easy comparison:

https://github.com/getsentry/raven-node/blob/master/lib/parsers.js#L154-L162